linker/linker_test.go

```diff
@@ -96,8 +114,12 @@ func TestLinkerValidation(t *testing.T) {
 	t.Parallel()
 	testCases := map[string]struct {
 		input map[string]string
+		// The correct order of passing files to protoc in command line
+		inputOrder []string
 		// Expected error message - leave empty if input is expected to succeed
-		expectedErr string
+		// This is set when the order of input file matters for protoc
```

Comment:
I think this comment belongs above, next to `inputOrder`, right?

Reply:
yeah I think Goland messed this up for me. I need to reset some settings since it now fights with me all the time.